### ISSUE TITLE
Font size doesn't enlarge text beyond 18

### ISSUE THREAD

**Issue by henryman93** [ElvUI-WotLK/ElvUI#621]
**Description**

Font size slider does not affect text beyond size 18, so anything from size 18 to 32 has the same font size.

**Version Number**

6.01

**Steps to Reproduce**

1. Move any font size slider (e.g. Unitframe) to any number between 18-32

**Expected Behavior**

Scale text according to font size selected

**Actual Behavior**

Font size 18 and above is the same

**Only ElvUI Enabled**

Yes

**Blizzard Issue?**

No

**Additional Information**

**Error Log**

```

```

**Comment by Bunny67** [comment]
Can not reproduce

18
![image](https://user-images.githubusercontent.com/5816685/64627535-7e8c8800-d3f8-11e9-95b6-094a5015dfcd.png)

32
![image](https://user-images.githubusercontent.com/5816685/64627597-9532df00-d3f8-11e9-94cb-d47914e235c8.png)

**Comment by henryman93** [comment]
Further troubleshooting showed this bug to be tied to the set UI Scale; the bigger the UI scale, the less the font size is allowed to grow. Let me know if you can now reproduce (playing native res 4K windowed borderless maximized however I changed resolution and problem remained, so I ruled graphics out of the troubleshoot)

Examples:
**UI scale set to 0.8**
Font size
6:
![image](https://user-images.githubusercontent.com/10605951/64629207-affe5700-d3ea-11e9-9b6c-a35a79a78187.png)

14 (threshold for this UI scale):
![image](https://user-images.githubusercontent.com/10605951/64629107-83e2d600-d3ea-11e9-990c-8c861039468b.png)

32:
![image](https://user-images.githubusercontent.com/10605951/64629166-9ceb8700-d3ea-11e9-97f5-f571b0b6508b.png)


**UI scale set to 0.4**
6:
![image](https://user-images.githubusercontent.com/10605951/64629361-f784e300-d3ea-11e9-9727-f437558f47e1.png)

14:
![image](https://user-images.githubusercontent.com/10605951/64629480-2ef38f80-d3eb-11e9-9d72-01ae1e160972.png)


28 (threshold for this UI scale):
![image](https://user-images.githubusercontent.com/10605951/64629400-09ff1c80-d3eb-11e9-9923-4b2817d8e910.png)

32:
![image](https://user-images.githubusercontent.com/10605951/64629445-1be0bf80-d3eb-11e9-9a11-38f483f89ec4.png)

**Comment by Bunny67** [comment]
I have no idea how this can be fixed.
This is similar to the error of WoW.
Client not change font size> 32 in full hd resolution
Perhaps there are restrictions for 4K resolution

**Comment by Bunny67** [comment]
API for Font
![image](https://user-images.githubusercontent.com/5816685/64634840-d205d280-d406-11e9-8759-bc891551a158.png)
if SetFont it doesn't apply size, then it’s impossible to apply otherwise

**Comment by henryman93** [comment]
It really seems to be a 4K issue with client. I've tested other computer with lower res, all font sizes works. Tested 4K computer with lower res and disabled upscaling on nvidia control panel, all font sizes are working. As soon as I crank it up to 4K (independent of Fullscreen of Windowed), fontsize gets capped (only UI scaling seems to affect cap). It is also independent of ElvUI (tested previous version and DBM, same issue); Weakauras worked but I believe the font scaling is different (and honestly pointless with all that pixelation).
So if there's no workaround, I leave to your discretion to close the issue. Thanks for helping with this and sorry for the wasted time 👍
